test/utils/create-tls-certificates.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash -e
+
+SERVER_CERT_FILE=server.cert
+SERVER_KEY_FILE=server.key
+SERVER_CSR_FILE=server.csr
+CA_CERT_FILE=ca.cert
+CA_KEY_FILE=ca.key
```

Comment:
I suggest putting all these files in a directory to easily remove them during test cleanup.